File: src/constants/enums.ts
export enum Game {
  GENSHIN = 'genshin',
  STARRAIL = 'starrail',
  ZZZ = 'zzz'
}

export enum DatabaseKey {
  GENSHIN = 'genshin',
  STARRAIL = 'starrail',
  ZZZ = 'zzz'
}

export const GAME_THUMBNAILS = {
  [Game.GENSHIN]: 'genshin-impact.png',
  [Game.STARRAIL]: 'star-rail.png',
  [Game.ZZZ]: 'zenless-zone-zero.png'
};

export const HOYO_REDEEM_URLS = {
  [Game.GENSHIN]: 'https://genshin.hoyoverse.com/gift?code=',
  [Game.STARRAIL]: 'https://hsr.hoyoverse.com/gift?code=',
  [Game.ZZZ]: 'https://zzz.hoyoverse.com/gift?code='
};

File: zenox/static/enums.py
from enum import Enum, StrEnum, IntEnum
from typing import Final

class PrintColors(StrEnum):
    HEADER = "\033[95m"
    OKBLUE = "\033[94m"
    OKCYAN = "\033[96m"
    OKGREEN = "\033[92m"
    WARNING = "\033[93m"
    FAIL = "\033[91m"
    ENDC = "\033[0m"
    BOLD = "\033[1m"
    UNDERLINE = "\033[4m"

class Game(StrEnum):
    GENSHIN = "Genshin Impact"
    STARRAIL = "Honkai: Star Rail"
    ZZZ = "Zenless Zone Zero"

class GenshinCity(StrEnum):
    MONDSTADT = "Mondstadt"
    LIYUE = "Liyue"
    INAZUMA = "Inazuma"
    SUMERU = "Sumeru"
    FONTAINE = "Fontaine"
    NATLAN = "Natlan"